lib/reek/context/code_context.rb

```diff
@@ -22,8 +22,8 @@ class CodeContext
       delegate %i(name type) => :exp
       delegate %i(visibility visibility= non_public_visibility?) => :visibility_tracker
 
-      attr_reader :exp, :children, :visibility_tracker, :statement_counter
-      private_attr_reader :context, :refs
+      attr_reader :context, :children, :exp, :visibility_tracker, :statement_counter
```

Comment:
Trivial, but as long as we’re touching this line anyway: can we order the attrs alphabetically? It’d be easier to scan when checking for, say, `:statement_counter`.